### PR TITLE
Remove EJB compatability tests for legacy (from before Jakarta EE 8) deployment descriptors 

### PR DESCRIPTION
**Pull request by scottmarlow**
For https://github.com/eclipse-ee4j/jakartaee-tck/issues/501

### CODE REVIEW

**Comment by scottmarlow** [comment]
Please do not merge before September 16 as discussed on Platform TCK mailing list.

**Comment by scottmarlow** [comment]
@alwin-joseph @RohitKumarJain @gurunrao please do not merge before September 16, 2020 (I can merge it then after one of you have reviewed the change).

**Review by RohitKumarJain — APPROVED** [review]
LGTM

**Pull request merged**
